test/e2e/features/config.feature

```diff
@@ -152,6 +152,21 @@ Feature: Test configuration settings
         And executing crc setup command succeeds
         Then stderr should not contain "Skipping above check"
 
+    Scenario: CRC config set and get preset property (default case) 
+        When setting config property "preset" to value "<preset-value>" succeeds
+        And "JSON" config file "crc.json" in CRC home folder does not contain key "preset"
+        When getting config property "preset" succeeds
+        And stdout should contain "Configuration property 'preset' is not set"
+        And stdout should contain "<preset-value>"
+        When unsetting config property "preset" succeeds
+        And stdout should contain "Successfully unset configuration property 'preset'"
+        And "JSON" config file "crc.json" in CRC home folder does not contain key "preset"
+        
+        @linux @darwin @windows 
+        Examples: Config property preset setting default
+            | preset-value | 
+            | openshift | 
```

Comment:
As default means only one option, may we can add it to the sentence instead of using the `Examples` feature?

Another option would be to unified both scenarios as most of the steps are the same and include the condition for those which are affected in case the preset is the default?  I think you can extend the Example table to :

|preset-value| default |
| openshift    | yes        |
| microshift   | no         |
| okd               | no         |

And use that default value to check on those conditions

WDYT?